src/SFA.DAS.AANHub.Application/CalendarEvents/Queries/GetCalendarEvent/GetCalendarEventByIdQueryResult.cs

```diff
@@ -0,0 +1,59 @@
+﻿using SFA.DAS.AANHub.Application.Models;
+using SFA.DAS.AANHub.Domain.Entities;
+
+namespace SFA.DAS.AANHub.Application.CalendarEvents.Queries.GetCalendarEvent;
+
+public class GetCalendarEventByIdQueryResult
+{
+    public Guid CalendarEventId { get; set; }
+    public int CalendarId { get; set; }
+    public string EventFormat { get; set; } = string.Empty;
+    public DateTime StartDate { get; set; }
+    public DateTime EndDate { get; set; }
+    public string Description { get; set; } = string.Empty;
+    public string? Summary { get; set; }
+    public int? RegionId { get; set; }
+    public string? Location { get; set; }
+    public string? Postcode { get; set; }
+    public double? Latitude { get; set; }
+    public double? Longitude { get; set; }
+    public string? EventLink { get; set; }
+    public string ContactName { get; set; } = string.Empty;
+    public string ContactEmail { get; set; } = string.Empty;
+    public bool IsActive { get; set; }
+    public string? CancelReason { get; set; } = null!;
+
+
+    public List<AttendeeModel> Attendees { get; set; } = new();
+    public List<EventGuestModel> EventGuests { get; set; } = new();
+
```

Comment:
All these fields look exactly the same as CalendarEvent? Can this inherit from CalendarEvent instead? (or maybe both based on common parent)? Have a look, I do know that sometimes this causes implicit operators and other mappers to fall apart though, so use your judgement to decide whether to address or not.